molgenis-data-elasticsearch/pom.xml

```diff
@@ -26,16 +18,40 @@
         </dependency>
         <dependency>
             <groupId>org.molgenis</groupId>
-            <artifactId>molgenis-data</artifactId>
+            <artifactId>molgenis-data-index</artifactId>
             <version>${project.version}</version>
-            <type>test-jar</type>
-            <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.molgenis</groupId>
             <artifactId>molgenis-jobs</artifactId>
             <version>${project.version}</version>
```

Comment:
Yes, but currently both IndexConfig (should be moved to molgenis-data-index) and EmbeddedElasticSearchServiceFactory (can be removed once we use standalone Elasticsearch) depend on it. Lets remove the dependency once we wrap up M4667).